lib/templates/immutable.yaml.mustache

```diff
@@ -12,6 +12,7 @@
          instance_type="{{=<% %>=}}{{ instance_type }}<%={{ }}=%>"
          keypair='{{{ keypair }}}'
          instance_tags='{"Environment":"{{{ environment }}}","Class":"{{{ className }}}-immutable","Name":"{{{ appName }}} (immutable)"}'
+         instance_tags='{"Environment":"environment","Class":"class-immutable","Name":"{{{ appName }}} (immutable)"}'
```

Comment:
we can lose class and environment entirely. Something like:

`instance_tags='{"Name":"{{{ appName }}} (immutable)"}'`